src/Robot.elm

```diff
@@ -7,17 +7,14 @@ import Html.Attributes as Attributes exposing (style)
 import Html.Events exposing (onClick)
 import Json.Decode
 
-main =
-    Browser.element { init = init, update = update, view = view, subscriptions = subscriptions }
-
 
 type alias Model =
-    { xblue : Int, yblue : Int , xred : Int, yred : Int}
+    { x : Int, y : Int, color : String}
 
 
-init : () -> ( Model, Cmd Msg )
-init _ =
-    ( { xblue = 50, yblue = 50 ,xred = 10, yred = 10}, Cmd.none )
+init : Int -> Int -> String -> Model
+init newX newY newColor =
+    { x = newX, y = newY, color = newColor}

```

Comment:
You can use the implicit constructor you create with `type alias`:
```elm
init : Int -> Int -> String -> Model
init x y color =
    Model x y color
```
or even (in a point free style 😮)
```elm
init : Int -> Int -> String -> Model
init =
    Model
```